.eslintrc.js

```diff
@@ -99,6 +101,14 @@ module.exports = {
             allowModules: ["xrpl-local"],
           },
         ],
+
+        // Tests are already in 2 callbacks, so max 3 is pretty restrictive
+        "max-nested-callbacks": [
```

Comment:
These rules have been merged in, you should be able to rebase for the updated versions